components/ChoreWeight.tsx

```diff
@@ -48,15 +57,19 @@ export default function ChoreWeight({
           <Card.Actions>
             <View style={s.content}>
               <View style={s.textContainer}>
-                <Text style={{ fontWeight: 'bold', fontSize: 18 }}>
+                <Text
+                  style={{ fontWeight: 'bold', fontSize: 18, color: textColor }}
+                >
                   Value:{' '}
                 </Text>
-                <Text>How energy-demanding is the task?</Text>
+                <Text style={{ color: textColor }}>
+                  How energy-demanding is the task?
+                </Text>
```

Comment:
Theme takes care of this
```suggestion
                <Text>
                  How energy-demanding is the task?
                </Text>
```